### ISSUE TITLE
Expected RBRACE when using angle 'turn' units with rotate

### ISSUE THREAD

**Issue by edmorley** [CSSLint/csslint#732]
### STR:
1. Visit http://csslint.net/
2. Enter:
```css
@keyframes action-bar-spin {
  0% { transform: rotate(0turn); }
  100% { transform: rotate(1turn); }
}
```

### Expected:
No errors

### Actual:
```
  | 2 | 26 | Parsing Errors | Expected RBRACE at line 2, col 26.  0% { transform: rotate(0turn); } | All
  | 3 | 28 | Parsing Errors | Expected RBRACE at line 3, col 28.  100% { transform: rotate(1turn); } | All
```

It seems like #662 / CSSLint/parser-lib#217 might have regressed?

Many thanks :-)
